scripts/typescript_checks.py

```diff
@@ -151,6 +151,7 @@
     'core/templates/pages/exploration-player-page/services/question-player-engine.service.ts',
     'core/templates/pages/facilitator-dashboard-page/facilitator-dashboard-page.import.ts',
     'core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.spec.ts',
+    'core/templates/pages/learner-dashboard-page/new-learner-dashboard.component.spec.ts',
```

Comment:
Do not add files here. Please see the "NOTE TO DEVELOPERS" on top.

Reply:
Thanks for pointing out. I have created a new file for the learner dashboard named **new-leaner-dashboard**. I have created this file so that I can manage the old and new dashboard. I have just copy the code of **learner-dashboard.component.spec.ts** to ** new-learner-dashboard.component.spec.ts** and made changes. When I run the strict typescripts test, it is giving me an error logs due to the code of the old learner-dashboard file, so I have added this file to this list. Once this project is complete. I will shift the code from `new-learner-dashboard.component.spec.ts` to `learner-dashboard-page.component.spec.ts` and remove the `new-learner-dashboard-page.component.spec.ts`
Thanks!

Reply:
I'm okay with this if @kevintab95 approves.

@SubhashThenua please be more careful and precise with your comments in the future. You refer to `learner-dashboard.component.spec.ts`, but I think you actually mean `learner-dashboard-page.component.spec.ts`. Similarly, you mention removing `new-learner-dashboard-page.component.spec.ts` when I think you mean `new-learner-dashboard.component.spec.ts`. These differences may seem trivial to you since you work with the frontend code a lot, but they're confusing for reviewers like me who don't work in the frontend much